test/tests/d91_rom/tests

```diff
@@ -0,0 +1,28 @@
+[Tests]
+  [./rom]
+    issues = '#14046'
+    design = 'D91LAROMANCEStressUpdate.md'
+    requirement = 'The system shall compute a creep rate based on a reduced order model for D91'
+    [./verification]
+      type = 'CSVDiff'
+      input = 'verification.i'
+      csvdiff = 'verification_out.csv'
+      cli_args = 'Executioner/dt=25 Executioner/num_steps=10'
+      petsc_version = '>=3.9.0'
+      max_parallel = 1
+      detail = 'in isolation (i.e. without a full displacement solve), and match with code-to-code comparison.'
+    [../]
+    [./verification_heavy]
+      type = 'CSVDiff'
+      input = 'verification.i'
+      csvdiff = 'verification_heavy_out.csv'
```

Comment:
@tophmatthews would you send us the code comparison csv file match for this heavy test please?

Reply:
This is the same as the non-heavy test `p91_verification_data.csv`, just more lines are sampled

Reply:
It's highly possible I didn't include the `verification_heavy_out.csv` gold file though. You can generate this through the test, and "gold" it by verifying the maximum diff is low (1e-6 or 1e-5 at worst?)

Reply:
I'll do this once I get the easy test working.